test/numbers.jl

```diff
@@ -988,6 +988,16 @@ end
     @test Float64(UInt128(3.7e19)) == 3.7e19
     @test Float64(UInt128(3.7e30)) == 3.7e30
 end
+@testset "Float16 vs Int comparisons" begin
+    @test !(Inf16 == typemax(Int16))
```

Comment:
Use `!=` here, probably.   This falls back to the definition `!=(x, y) = !(x == y)` so it will do what you want.

(The `@test` macro has special handling for `@test A <comparison> B` to print nicer failure messages.)

Reply:
I fixed this Steven. Thanks.